### ISSUE TITLE
How to get filterbanks.npy

### ISSUE THREAD

**Issue by GauriDhande** [carlini/audio_adversarial_examples#36]
How to save filterbanks.npy? tf_logits.py loads the filterbanks.npy directly. Where were they saved?

**Comment by dijksterhuis** [comment]
`filterbanks.npy` lives in the repo itself. `tf_logits.py` should just load it straight from there if you've cloned the repo.

https://github.com/carlini/audio_adversarial_examples/blob/master/filterbanks.npy

**Comment by GauriDhande** [comment]
Hi @dijksterhuis, I would like to use the Google Speech Commands dataset. So, I will like to know if these filterbanks be applicable for Google Speech Commands also?

**Comment by carlini** [comment]
These filterbanks are designed to replicate the mozilla deepspeech results, independent of the underlying dataset.

That said, if you try to run this model on a new dataset, it's likely that there will be some form of domain shift and therefore a drop in accuracy.
